app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -57,9 +57,9 @@ def info
 
       def new
         @page ||= Page.new(layoutpage: params[:layoutpage] == "true", parent_id: params[:parent_id])
-        @page_layouts = PageLayout.layouts_for_select(@current_language.id, @page.layoutpage?)
+        @page_layouts = PageLayout.layouts_for_select(@current_language.id, @page.layoutpage?, @page.parent &.page_layout)
```

Comment:
Shouldn't this be?

```suggestion
        @page_layouts = PageLayout.layouts_for_select(@current_language.id, @page.layoutpage?, @page.parent&.page_layout)
```

A test would have could this. Could you please add one?